tools/ci_static_check_reports/unit_tests/test_build_static_checkers_reports.py

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+import os
+
+from ..build_static_checkers_reports import build_static_checkers_reports
+
+
+def test_build_static_checkers_reports() -> None:
+    changed_module_path = "tools/ci_static_check_reports"
+    build_static_checkers_reports([changed_module_path])
+    static_checker_reports_path = "static_checker_reports/tools/ci_static_check_reports"
```

Comment:
can we test the following cases?
1) it is not python code
2) module is not exist

Reply:
sure, will do it